src/main/java/ay2122s1_cs2103t_w16_2/btbb/model/order/UniqueOrderList.java

```diff
@@ -49,6 +50,20 @@ public void setOrder(Order target, Order editedOrder) throws NotFoundException {
         internalList.set(index, editedOrder);
     }
 
+    /**
+     * Removes the equivalent order from the list of orders.
+     * The ingredient must exist in the list.
+     *
+     * @param toRemove The order to remove from the list.
+     * @throws NotFoundException when there is no equivalent order found in the list.
+     */
+    public void remove(Order toRemove) throws NotFoundException {
+        requireNonNull(toRemove);
+        if (!internalList.remove(toRemove)) {
+            throw new NotFoundException(Ingredient.class.getName());
```

Comment:
Could you put this above setOrder?
```suggestion
            throw new NotFoundException(Order.class.getName());
```